charts/k10/k10/4.5.6/files/kasten-logo.svg

```diff
@@ -0,0 +1,24 @@
+<svg viewBox="0 0 986 215" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Where will this logo be displayed? 
Is this the one with `Kasten By Veeam` or just `Kasten`? Is there a favicon with `By Veeam` available?

Reply:
this logo is displayed on the 
1. Charts page on Marketplace 
![image](https://user-images.githubusercontent.com/41359156/149274420-8e3f0ce9-383b-4472-8578-0f2a0e0e1908.png)


2. On Application's description page 

![image](https://user-images.githubusercontent.com/41359156/149274602-5ed5fee7-d9ab-4174-a567-408715b71cba.png)


 And this information is loaded from Icon field. I will update it to use `kasten by veaam` See my comments here [https://github.com/kastenhq/partner-charts/pull/1#discussion_r783650784](https://github.com/kastenhq/partner-charts/pull/1#discussion_r783650784)